source/agora/consensus/SCPEnvelopeStore.d

```diff
@@ -131,18 +132,19 @@ public class SCPEnvelopeStore
 
     ***************************************************************************/
 
-    public int opApply (scope int delegate(const ref SCPEnvelope) dg)
+    public int opApply (scope int delegate(bool processed, const ref SCPEnvelope) dg)
     {
         () @trusted
         {
             auto results = this.db.execute(
-                "SELECT envelope FROM scp_envelopes");
+                "SELECT envelope, processed FROM scp_envelopes");
 
             foreach (ref row; results)
             {
                 auto env = deserializeFull!(const SCPEnvelope)(row.peek!(ubyte[])(0));
+                auto proc = row.peek!bool(1);
```

Comment:
IMO `processed` is better as variable name.

Reply:
Isn't `processed` already used above?

Reply:
Done.